Sources/ElasticSwift/ElasticSwift.swift

```diff
@@ -138,11 +150,18 @@ public class ESClient {
         self.transport = Transport(forHosts: hosts, credentials: credentials, sslConfig: sslConfig)
     }
     
-    func execute(request: Request, completionHandler: @escaping (_ response: ESResponse) -> Void) -> Void {
+    func execute<R : Request>(request: R, completionHandler: @escaping (_ response: ESResponse) -> ()) throws {
         if request.method == .GET {
-            self.transport.performRequest(method: request.method, endPoint: request.endPoint, params: [], completionHandler: completionHandler)
+            self.transport.performRequest(method: request.method, endPoint: request.endPoint, params: request.parameters, completionHandler: completionHandler)
```

Comment:
Parameters in GET operation are now supported

Reply:
As far as I know swift-nio is more useful as server side development and this is a client component. 
For iOS development it would be an error to limit iOS 12+ by using swift-nio with very limited advantages: average apps are supporting last 3 iOS versions, this means that this lib cannot be used before iOS 14 (2 years from now).
If you decide to do so, I’ll be forced to keep a dedicated iOS fork.

Reply:
While publishing `alpha-5` it was throwing an error because of swift 5.0. `swift-nio` changes are not merged to dev & master. With that said `swift-nio` will not be a problem because it's a plugable and there will be a `URLSession` based implementation for platforms where `swift-nio` is not available. `feature/modules` branch has these changes. I will fix all the compatibility issues before merging to `master`.